pkg/util/kubernetes/clustername/clustername.go

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package clustername
+
+import (
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+var (
```

Comment:
Although we're gonna have a single instance, I'd be more comfortable with these being grouped in a `struct`, and `GetClusterName()` being a simple `return mytest.getClusterName()`. this way, you can delete `ResetClusterName` and test class instances directly

Reply:
Thanks for the tips! 👍